README.md

```diff
@@ -103,13 +103,13 @@ set -g nvm_mirror https://npm.taobao.org/mirrors/node
 
 ## `.nvmrc`
 
-An `.nvmrc` file makes it easy to peg a specific version of Node for different projects. Just create an `.nvmrc` file containing a Node version number or alias, e.g., `node`, `lts`, `carbon`, etc., in the root of your project.
+An `.nvmrc` file makes it easy to peg a specific version of Node for different projects. Just create an `.nvmrc` (or `.node-version`) file containing a version number or alias, e.g., `latest`, `lts`, `carbon`, in the root of your project.
 
 ```console
 node -v >.nvmrc
 ```
 
-Then run `nvm install` to install and activate that version. This will traverse the directory hierarchy looking for the nearest `.nvmrc` file.
+Then run `nvm install` to install and activate that version. Works from anywhere inside your project by traversing the directory hierarchy until a `.nvmrc` is found.
 
```

Comment:
until **an** `.nvmrc`

Reply:
(suppose it depends if you pronounce the dot or not, but you used "an" elsewhere so just mentioned this for consistency.